<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QHexViewWidget</class>
 <widget class="QWidget" name="QHexViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonGoTo">
       <property name="text">
        <string>Go to address</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelCursor">
       <property name="text">
        <string>Cursor:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEditHEX" name="lineEditCursorAddress">
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelSelection">
       <property name="text">
        <string>Selection:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEditHEX" name="lineEditSelectionAddress">
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelSize">
       <property name="text">
        <string>Size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEditHEX" name="lineEditSelectionSize">
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxReadonly">
       <property name="text">
        <string>Readonly</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QHexView" name="scrollAreaHex">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>840</width>
        <height>544</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QHexView</class>
   <extends>QScrollArea</extends>
   <header>qhexview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QLineEditHEX</class>
   <extends>QLineEdit</extends>
   <header>qlineedithex.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
